panic: open {home}/.liferay/cli/cli-releases.json: no such file or directory on first execution of cli

```
panic: open {home}/.liferay/cli/cli-releases.json: no such file or directory

goroutine 1 [running]:
liferay.com/liferay/cli/cmd.checkForUpdate()
	liferay.com/liferay/cli/cmd/liferay.go:71 +0x354
liferay.com/liferay/cli/cmd.initConfig()
	liferay.com/liferay/cli/cmd/liferay.go:146 +0x350
liferay.com/liferay/cli/cmd.init.0()
	liferay.com/liferay/cli/cmd/liferay.go:88 +0x1d
```

--- a/http/http.go
+++ b/http/http.go
@@ -5,6 +5,7 @@ import (
 	"io/ioutil"
 	"net/http"
 	"os"
+	"path/filepath"
 
 	"github.com/spf13/viper"
 )
@@ -53,6 +54,11 @@ func GetOrFetchBytes(options GetOrFetchBytesOptions) ([]byte, error) {
 		viper.WriteConfig()
 
 		body, err := ioutil.ReadAll(resp.Body)
+		if err != nil {
+			return nil, err
+		}
+
+		err = os.MkdirAll(filepath.Dir(viper.GetString(options.FileKey)), 0775)
 		if err != nil {
 			return nil, err
 		}

